issues compiling with go from source

bk@sneak:~/Downloads/dstat-graph-1.0.0$ make pkg
go get gopkg.in/urfave/cli.v1
go get github.com/comail/colog
go get github.com/wcharczuk/go-chart
go generate
mkdir -p pkg
GOOS=linux GOARCH=amd64 go build -o pkg/dstat-graph_linux_amd64-1.0.0 cmd/main.go
# github.com/octu0/dstat-graph
/usr/local/go/src/github.com/octu0/dstat-graph/linechart.go:75:9: unknown field 'Show' in struct literal of type chart.Style
/usr/local/go/src/github.com/octu0/dstat-graph/linechart.go:92:14: undefined: chart.StyleShow
/usr/local/go/src/github.com/octu0/dstat-graph/linechart.go:98:23: undefined: chart.StyleShow
/usr/local/go/src/github.com/octu0/dstat-graph/linechart.go:110:9: unknown field 'Show' in struct literal of type chart.Style
make: *** [Makefile:36: pkg-linux-amd64] Error 2

--- a/README.md
+++ b/README.md
@@ -23,6 +23,10 @@ $ dstat-graph --csv ./dstat.csv -o cpu.png -f usr,sys,idl,wai
 
 see more [examples](https://github.com/octu0/dstat-graph/tree/master/examples).
 
+## Download
+
+Linux amd64 / Darwin amd64 binaries are available in [Releases](https://github.com/octu0/dstat-graph/releases)
+
 ## Build
 
 Build requires Go version 1.11+ installed.

--- a/examples/README.md
+++ b/examples/README.md
@@ -1,0 +1,20 @@
+# example
+
+To run example, you need to install dstat-graph in `$GOPATH`
+
+```
+go get github.com/octu0/dstat-graph
+```
+
+Next, generate a graph using csv in the example directory
+
+```
+$ cd examples
+$ go run $GOPATH/src/github.com/octu0/dstat-graph/cmd/main.go --csv ./dstat.csv -o cpu.png -f usr,sys,idl,wai
+```
+
+You can check the generated image using the viewer app.
+
+![cpu](https://github.com/octu0/dstat-graph/blob/master/examples/cpu.png?raw=true)  
+
+useful as [dstat2graph.sh](https://github.com/octu0/dstat-graph/blob/master/examples/dstat2graph.sh)

--- a/linechart.go
+++ b/linechart.go
@@ -72,7 +72,6 @@ func (c *LineChart) Read(r io.Reader) error {
     series = append(series, chart.TimeSeries{
       Name: col,
       Style: chart.Style{
-        Show: true,
         StrokeColor: color,
         FillColor:   color.WithAlpha(15),
       },
@@ -89,13 +88,11 @@ func (c *LineChart) Read(r io.Reader) error {
       },
     },
     YAxis: chart.YAxis{
-      Style: chart.StyleShow(),
       ValueFormatter: func(v interface{}) string {
         return fmt.Sprintf("%3.2f", v.(float64))
       },
     },
     XAxis: chart.XAxis{
-      Style:          chart.StyleShow(),
       ValueFormatter: func(v interface{}) string {
         format := "01-02 03:04:05"
         if t, isTime := v.(time.Time); isTime {
@@ -107,7 +104,6 @@ func (c *LineChart) Read(r io.Reader) error {
         return fmt.Sprintf("<unknown_axis>: %#v", v)
       },
       GridMajorStyle: chart.Style{
-        Show:        true,
         StrokeColor: chart.ColorAlternateGray,
         StrokeWidth: 2.0,
       },

--- a/version.go
+++ b/version.go
@@ -2,6 +2,6 @@ package graph
 
 const (
   AppName string = "dstat-graph"
-  Version string = "1.0.0"
+  Version string = "1.0.1"
   UA      string = AppName + "/" + Version
 )

